headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -864,6 +864,21 @@ struct tile {
 };
 ASSERT_SIZE(struct tile, 20);
 
+// Information about the rooms on the current floor
+struct room_data {
+    uint8_t room_id;
+    undefined field_0x1;                 // Initialized to 0
+    struct position bottom_right_corner; // 0x2
+    struct position top_left_corner;     // 0x6
+    undefined field_0xA;                 // Doesn't get initialized, likely padding
```

Comment:
Minor nitpick, but the `field_0x...` is a Ghidra convention, and it uses lowercase 0xa-0xf. I'll go ahead and make the tweak.